pkgs/build-support/cc-wrapper/setup-hook.sh

```diff
@@ -97,10 +97,12 @@ ccWrapper_addCVars () {
 # setup-hook, which `role` tracks.
 if [ -n "${crossConfig:-}" ]; then
     export NIX_CC_WRAPPER_@infixSalt@_TARGET_BUILD=1
-    role="BUILD_"
+    role='BUILD_'
+    role_post='_FOR_BUILD'
 else
     export NIX_CC_WRAPPER_@infixSalt@_TARGET_HOST=1
-    role=""
+    role=''
+    role_post=''
```

Comment:
We could probably make this `'_FOR_HOST`, but I doubt that's necessary / I don't particularly want to deal with more native breakage.